lib/agent/https_proxy_ocsp_agent.js

```diff
@@ -98,7 +98,25 @@ function connect(req, opts, fn)
 {
   var proxy = this.proxy;
   var agent = this;
-  Logger.getInstance().debug("Using proxy=%s for host %s", proxy.host, opts.host);
+
+  // try and detect if there's proxy envvar set, currently it's not logged anywhere
+  if (process.env.HTTP_PROXY || process.env.HTTPS_PROXY) {
+    envHttpProxy = process.env.HTTP_PROXY ? 'HTTP_PROXY: ' + process.env.HTTP_PROXY : 'HTTP_PROXY: <unset>'
+    envHttpsProxy = process.env.HTTPS_PROXY ? 'HTTPS_PROXY: ' + process.env.HTTPS_PROXY : 'HTTPS_PROXY: <unset>'
+    envNoProxy = process.env.NO_PROXY ? 'NO_PROXY: ' + process.env.NO_PROXY : 'NO_PROXY: <unset>'
+    Logger.getInstance().debug('PROXY environment variables: %s %s %s'
+        , envHttpProxy, envHttpsProxy, envNoProxy)
+  }
+
+  // log proxy host, port, and user for observability
+  let prxPort = proxy.port ? ':' + proxy.port : ''
```

Comment:
let's prefer `const`s where it is possible

Reply:
ok